pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.bacon.pnc;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jboss.pnc.bacon.common.cli.AbstractGetSpecificCommand;
+import org.jboss.pnc.bacon.pnc.common.ClientCreator;
+import org.jboss.pnc.client.ClientException;
+import org.jboss.pnc.client.OperationClient;
+import org.jboss.pnc.dto.DeliverableAnalyzerOperation;
+import picocli.CommandLine.Command;
+
+@Slf4j
+@Command(name = "operation", description = "Long running operations", subcommands = { OperationCli.Get.class })
+public class OperationCli {
```

Comment:
I don't know, looks like part of deliverable analysis, either we can create separate command group for all deliverable commands or put this into product milestone where rest of them lives. @thescouser89 ?

Reply:
there will be other operations in the future, but I'm fine if you find some nice place for it

Reply:
you mean other than ProductMilestoneCli where rest of them are?

Reply:
yeah @jomrazek , a separate group for deliverables would be nice. Same as for DA.